docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,279 @@
+
+For Fuji Testnet, 5 is the correct value to use.
+
+```js
+const networkID: number = 5;
+```
+
+To learn more about encoded addresses, click [here](../apis/avalanchejs/manage-x-chain-keys.md#encode-bech32-addresses).
+
+**Set the Correct Amount To Send:**
+
+By default the scripts send the wallet's entire AVAX balance:
+
+```js
+const balance: BN = new BN(getBalanceResponse.balance);
+const amount: BN = balance.sub(fee);
+```
+
+To send a different amount, please replace the code above with the following. Below sets a new value of 0.01 AVAX (`10000000` GWEI). Value is set in GWEI format where `1,000,000,000` GWEI = 1 AVAX
+
+```js
+const value: BN = new BN("10000000");
+const amount: BN = value.sub(fee);
+```
+
+:::tip
+Snowtrace provides a [unit converter](https://snowtrace.io/unitconverter) between different units
+:::
+
+Run the export script:
+
+```sh
+$ ts-node examples/avm/buildExportTx-cchain-avax.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: Rgg2412kaczRYC3taasvG6bYoqG7tBQG6WfacNdumKDKsVWpF
+```
+
+#### Import the Avax Token From X-Chain to C-Chain
+
+Select the [**`examples/evm`**](https://github.com/ava-labs/avalanchejs/tree/master/examples/evm) folder to view the AvalancheJS C-Chain examples. To import AVAX to the C-Chain from the X-Chain, select [`evm/buildImportTx-xchain.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildImportTx-xchain.ts)
+
+Copy the [network setting from above](#network-setting) into `evm/buildImportTx-xchain.ts`.
+
+Navigate to this part of the code and ensure that the `cHexAddress`(_Your C-Chain wallet address_) and `private key` are correct:
+
+```ts
+const cHexAddress: string = "<YOUR-CCHAIN-WALLET-ADDRESS-HERE>";
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
+```
+
+Run the import script:
+
+```sh
+$ ts-node examples/evm/buildImportTx-xchain.ts
+```
+
+This returns:
+
+```sh
+Success! TXID: r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1
+```
+
+That's it! You've transferred AVAX from the X-Chain to C-Chain!
+
+#### Verify the Transaction
+
+You can now pass this txID `r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1` into [examples/avm/getTx.ts](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/getTx.ts), plus other similar network settings, then you can run
+
+```zsh
+$ ts-node examples/avm/getTx.ts
+```
+
+which returns:
+
+```js
+{
+  unsignedTx: {
+    networkID: 1,
+    blockchainID: '2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM',
+    outputs: [],
+    inputs: [ [Object] ],
+    memo: '0x',
+    destinationChain: '11111111111111111111111111111111LpoYY',
+    exportedOutputs: [ [Object] ]
+  },
+  credentials: [
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    }
+  ]
+}
+```
+
+### Transfer from the C-Chain to the X-Chain
+
+To return the AVAX back to the X-Chain, you need to do the transfer in the opposite direction.
+
+#### Export the Avax Token From C-Chain to X-Chain
```

Comment:
I now separated this section into two sections of `Export` and `Import`. Please update the doc following this structure instead of lumping everything in one section.

Reply:
roger